src/pages/billing/Project.js

```diff
@@ -391,6 +417,19 @@ const ProjectDetail = ({ billingProject, reloadBillingProject, billingAccounts,
             div({ style: { marginTop: '1rem' } },
               ['Note: Changing the billing account for this billing project will clear the workflow spend report configuration.'])
           ])])
+        ]),
+        showBillingRemovalModal && h(Modal, {
+          title: 'Remove Billing Account',
+          onDismiss: () => setShowBillingRemovalModal(false),
+          okButton: h(ButtonPrimary, {
+            onClick: () => {
+              setShowBillingModal(false)
+              removeBillingAccount(selectedBilling).then(reloadBillingProject)
```

Comment:
modal stays open when I click OK. I can click OK multiple times and it never goes away. loading dialogue opens in the background
![Screen Shot 2021-11-01 at 12 24 21 PM](https://user-images.githubusercontent.com/5375000/139706135-9e8df418-b17d-46f2-b877-86d0586e6883.png)